### ISSUE TITLE
Questions about Reproduction and Training Pipeline

### ISSUE THREAD

**Issue by Serious-Huang** [ZexiChen502/UniCure#4]
Hi,

About retrain the model from scratch and I have some questions regarding main.py.

1. In `train_lincs_step1`, is the file `lincs2020_control.parquet` generated by extracting control samples from `lincs2020_merge_cid.parquet`? (`lincs2020_control.parquet` is not provided)

2. The overall training workflow is not entirely clear to me. I would like to clarify a few issues that affect the reproducibility of the training pipeline. Files such as `./result/{seed}/lincs2020/best_stage_1_model.pth` are referenced in the code, but there is no clear description of how these files are generated. This makes it difficult to understand the dependencies between stages and to reproduce each training step independently.

If I want to retrain `UniCure` from scratch, would the correct procedure be something like the following?
```
if __name__ == '__main__':

    seed = 4
    accelerator = Accelerator()

    train_lincs_step1(4)
    train_lincs_step2(4)
    train_sciplex3(4)
    train_sciplex4(4)
    train_tahoe(seed=4)
```

Regarding `./requirement/model_weights/best_model.pth`: Is this model trained solely on the LINCS2020 dataset, or does it include fine-tuning on additional datasets?

These details are important for users who want to retrain UniCure from scratch or modify the training pipeline.

I would really appreciate any clarification on the intended training order and data dependencies between these steps.

Thank you again for your time and support.
